wormhole/dial.go

```diff
@@ -511,6 +511,9 @@ func Join(slot, pass string, sigserv string, pc *webrtc.PeerConnection) (*Wormho
 	// An unintended destination is likely going to fail PAKE.
 
 	msgA, pake, err := cpace.Start(pass, cpace.NewContextInfo("", "", nil))
+	if err != nil {
```

Comment:
FWIW, cpace.NewContextInfo cannot return any errors on static inputs like we use it here, but checking is good anyway.